### PR TITLE
CUMULUS-2345: Deploy ORCA with Cumulus

### PR DESCRIPTION
**Pull request by jennyhliu**
**Summary:** Summary of changes

Addresses [CUMULUS-2345: Deploy ORCA configuration via Terraform](https://bugs.earthdata.nasa.gov/browse/CUMULUS-2345)

## Changes

* Deploy ORCA with Cumulus via terraform
* Add `CopyToGlacier` step to example workflow IngestAndPublishGranule 
* There is no API to disable ORCA for a collection, so add meta.excludeFileTypes to collections for integration tests.
ORCA doesn't support multiple protected buckets, it will throw error when reading from 'protected-2' bucket.  There is a orca ticket https://bugs.earthdata.nasa.gov/browse/ORCA-140
* Add integration test to verify CopyToGlacier lambda output

## Update your cumulus stack:

Update example/cumulus-tf/terraform.tfvars

* add two variables and provide your own passwords
Password Constraints: At least 8 printable ASCII characters. Can't contain any of the following: / (slash), '(single quote), "(double quote) and @ (at sign).
database_app_user_pw = "password"
postgres_user_pw     = "password"

* Update lambda_subnet_ids to have two subnet ids
Subnet ids can be found under https://console.aws.amazon.com/vpc/home?region=us-east-1#subnets:sort=SubnetId
Use the Subnet IDs for "Private Application..."
lambda_subnet_ids = [id1, id2]

* Create a regular s3 bucket and add it to buckets configuration, see example/deployments/cumulus/sandbox.tfvars.
  glacier = {
    name = "my-orca-glacier-bucket"
    type = "glacier"
  },

## PR Checklist

- [x] Update CHANGELOG
- [ ] Unit tests
- [ ] Adhoc testing
- [x] Integration tests

### CODE REVIEW

File: example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js [hunk 1]
```diff
@@ -848,6 +875,17 @@ describe('The S3 Ingest Granules workflow', () => {
           );
           expect(reingestExecution.asyncOperationId).toEqual(asyncOperationId);
         });
+
+        // TODO remove the glacier files via ORCA API when the API is available
```
Comment (flamingbear) [inline example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js:879]: Same here, if there's a ticket, can we link it so we can check when this happens?

**Review by flamingbear — APPROVED** [review]
Looked it over, looks great.  I deployed the branch and saw the ORCA changes, ran the integration tests and saw the files backed up into the orca bucket and that they had storage classes of glacier.
I'm not sure who else wanted to look at this, but I can approve. 

Hopefully you can add the links to the ORCA tickets.

File: packages/integration-tests/Collections.js [hunk 2]
```diff
@@ -85,6 +85,9 @@ const buildRandomizedCollection = (overrides = {}) => ({
   granuleIdExtraction: '^([^.]+)\..+$',
   sampleFileName: 'asdf.jpg',
   url_path: randomId('url-path-'),
+  meta: {
+    excludeFileTypes: ['.cmr.json', '.cmr.xml', '.h5', '.hdf', '.jpg', '.md5', '.met', '.nc', '.txt'],
```
Comment (Jkovarik) [inline packages/integration-tests/Collections.js:89]: Why was this added?

Comment (jennyhliu) [reply packages/integration-tests/Collections.js:89]: There is no API to disable ORCA for a collection, so add meta.excludeFileTypes to collections for integration tests.
Some tests use the collection returned from buildRandomizedCollection/createCollection  for IngestAndPublishGranule workflow, and IngestAndPublishGranule workflow has CopyToGlacier step.
